### DISCUSSION TITLE
Make the kiai effect more customizable

### DISCUSSION THREAD

**Discussion by NovaliX-Dev** [ppy/osu#21333] [Ideas] [answered]
The kiai effect is an effect wildly use in osu!. From what i saw, it's used everywhere and mostly for music drops.
But there is something the kiai effect lacks of and this is controls and settings. We can't change anything in the kiai effect to customize it.

So here is what which settings i propose in the editor : 
- the color of the pulse (to give more controls over the theme of a map, like in ocean theme maps the lights can be more blueish)
- The frequency of the pulse only. I mean, we could do this already by changing the bpm, but for the mania mod this gets really painful : with the bpm varying, the speeds is changing constantly.
- Add the ability to make only one pulse and not pulse based bpm. This is useful for parts of music which are not sync at all.
- Add the ability to not show the kiai at all while being playing maps. This is not my idea, this is from @covelloz actually. This is not a customization, but i agree with him.

What do you think about all of this ?

**Comment by covelloz** [comment]
Would be cool to be able to disable kiai lighting effects altogether (if that is what you mean by intensity).
The flashing can cause seizures for those with photosensitive epilepsy.

It is a bit annoying to be blinded by flashing hit circles at (usually) the most intense portions of the beatmap. Apparently people have been asking for this feature for *years* in the forums. I noticed in lazer that the kiai flashing is more obvious than in stable for my skin.

**Reply by NovaliX-Dev** [reply]
Yeah, the lighting is what i've called the intensity (i've should called it pulse).

And yeah this is totally a thing to add. I'll modify my first message to add your idea.

**Comment by peppy** [comment, marked as answer]
Intensity being tracked by #22081.
